### ISSUE TITLE
Browsersync UI page not working for Laravel or Sage

### ISSUE THREAD

**Issue by Pondake** [BrowserSync/browser-sync#1891]
### Issue details

It seems that the UI page doesn't work on my latest projects. The page will partially load but gives JS errors. 

### Steps to reproduce/test case

1. Create a laravel app or bedrock installation + sage. 
2. Link valet (if using a Mac) or use something related to get a .test domain working. (Laragon for Windows)
3. If you're creating a laravel app, place this in webpack.mix.js

```
mix.browserSync({
    proxy: 'yourprojectname.test',
});
```

### Please specify which version of Browsersync, node and npm you're running

- Browsersync [  2.27.4  ]
- Node        [  15.14.0  ]
- Npm         [  7.19.1  ]

### Affected platforms

- [ ] linux
- [x] windows
- [x] OS X
- [ ] freebsd
- [ ] solaris
- [ ] other _(please specify which)_

### Browsersync use-case

- [x] API (I'm running it with laravel Mix in my latest use case)
- [ ] Gulp
- [ ] Grunt. 
- [ ] CLI

### for all other use-cases, (gulp, grunt etc), please show us exactly how you're using Browsersync

To get around using "artisan serve" a.k.a. 2 commands to develop my application. Everything works fine except the UI page which I use to example; enable remote debugging. 

![Schermafbeelding 2021-07-20 om 15 05 57](https://user-images.githubusercontent.com/4643209/126329173-1e9a4714-5ec2-44a0-be4a-85b5a412e699.png)

**Comment by Pondake** [comment]
Weird thing is that I have an older project which has the exact same package.json. Repopulating the node_modules folder by deleting and installing makes no difference. That older project has a working UI though.

I also reinstalled Node and tried different version, but it would make no sense that 1 project works and the other doesn't.
Even the .ENV file and webpack.mix.js file are (except for the URL and database) the same. I'm at a loss where to look atm.

**Comment by bfrickenstein** [comment]
Same problem ... and the website at port 3000 is empty.

**Comment by bfrickenstein** [comment]
Downgrade to 2.24.7 and it works.

**Comment by j0eii** [comment]
I confirm this issue exists, I have experienced the same issue.

**Comment by j0eii** [comment]
for laravel 8.40 or above,

you need to downgrade the browsersync to the following version in package.json for temporary fix:

```
        "browser-sync": "~2.24.*",
        "browser-sync-webpack-plugin": "~2.2.*",
```

then run `yarn install`

**Comment by itergator** [comment]
duplicate of #1885?

**Comment by sharkfin009** [comment]
> duplicate of #1885?

yes i think so

**Comment by shakyShane** [comment]
Fixed in browser-sync@2.27.5 :)
